halogen/schema.py

```diff
@@ -421,6 +439,8 @@ def serialize(cls, value, **kwargs):
             except (AttributeError, KeyError):
                 if attr.required:
                     raise
+            except ExcludedValueException:
+                pass
```

Comment:
maybe write a log line here?

Reply:
Expected case, not warning or error case. Possible could be a debug statement, but don't see the value of that too much.